src/Sylius/Behat/Context/Ui/CheckoutContext.php

```diff
@@ -96,15 +97,45 @@ public function iProceedSelectingOfflinePaymentMethod($paymentMethodName)
             'phoneNumber' => '321123456',
         ];
         $checkoutAddressingPage->fillAddressingDetails($addressingDetails);
-        $checkoutAddressingPage->pressButton('Continue');
+        $checkoutAddressingPage->forward();
 
         $checkoutShippingPage = $this->getPage('Checkout\CheckoutShippingStep');
-        $checkoutShippingPage->pressRadio('Free');
-        $checkoutShippingPage->pressButton('Continue');
+        $checkoutShippingPage->selectShippingMethod('Free');
 
         $checkoutPaymentPage = $this->getPage('Checkout\CheckoutPaymentStep');
-        $checkoutPaymentPage->pressRadio($paymentMethodName);
-        $checkoutPaymentPage->pressButton('Continue');
+        $checkoutPaymentPage->selectPaymentMethod($paymentMethodName);
+    }
+
+    /**
+     * @When /^I proceed selecting "([^"]+)" shipping method$/
+     * @Given /^I chose "([^"]*)" shipping method$/
```

Comment:
The only thing that is confusing now is that in a step `I chose X shipping method` I not only choose the method, but also fill in the addressing step, but I believe it can stay for now.